backend/src/BuiltinExecution/Libs/String.fs

```diff
@@ -421,6 +421,26 @@ let fns : List<BuiltInFn> =
       deprecated = NotDeprecated }
 
 
+    { name = fn "fromBytesRecoverable" 0
```

Comment:
I think this is backwards.

Errors will be ignored in the function that returns a String, and errors will not be ignored in the function that returns `Option<String>`.

I had intended fromBytesRecoverable to be the name for the function that returned a string (and hence recovers). But I think it's a poor name.

Reply:
Please correct me if I miss-understood what you mean here:
- The original fromBytes we had (which returns a String) should be renamed to something that indicates that it recovers)
- The newly added fromBytes (currently named fromBytesRecoverable) should be renamed to fromBytes
- We should then update all the references to the original 'fromBytes' method with its new name

Name suggestions:
- fromBytesWithReplacement
- fromBytesToSafeString
- fromBytesIgnoreErrors

Reply:
I like `fromBytesWithReplacement`

Yes, this sounds right. The default should be that the Bytes might not be strings and the user should opt into replacement.